apps/frontend/src/lib/landing/docs/docs.data.ts

```diff
@@ -192,4 +192,15 @@ export const sdkDocs: SDKSection[] = [
 		],
 		type: 'sdk',
 	},
+	{
+		id: 'ruby',
+		title: 'Ruby SDK',
+		docsPath: 'https://github.com/logdash-io/ruby-sdk?tab=readme-ov-file',
```

Comment:
I noticed that these docs have duplicated section `view` - could you please fix that?